Beginner/MultipleScheduledTasks/src/main/java/executor/MultipleSchedulerMain.java

```diff
@@ -0,0 +1,33 @@
+package executor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class MultipleSchedulerMain {
+
+    public List<Schedule> getSchedulerTasks() {
+        List<Schedule> schedulers = new ArrayList<>();
+        System.out.println("How many tasks do you want to schedule?");
+        Scanner sc = new Scanner(System.in);
+        int num = sc.nextInt();
+        for(int i = 0; i < num; i++) {
+            Schedule scheduler = new Schedule();
+            System.out.println("Enter your message:");
+            String message = sc.next();
```

Comment:
Can we use nextLine() here?

Reply:
nextLine() will place the cursor to the next line after reading the input. In this case, it impacts the reading of "interval" input.